internal/docker/ubuntu-22.04

```diff
@@ -0,0 +1,44 @@
+# syntax=docker/dockerfile:1
+
+# Select base image
+FROM ubuntu:22.04
+
+# Disable prompt during packages installation
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Minimize the number of layers (RUN, COPY and ADD create layers)
+# https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers
+RUN echo "#### Installing dependencies" \
+  && apt update && apt install -y \
```

Comment:
For the `elkodon_env` I had to install `gcc` as well for bindgen - but when it works then it works :/

Reply:
The build error message requested clang and after adding it to the image the build was successful